components/components.yaml

```diff
@@ -704,7 +704,7 @@
   Name: Particular.TimeoutMigration
   DocsUrl: nservicebus/tools
   GitHubOwner: Particular
-  Category: Persistence
+  Category: Tool
```

Comment:
Needs a PR to https://github.com/Particular/DocsEngine/blob/master/src/ParticularDocsCore/Component/ComponentCategory.cs to work

Reply:
https://github.com/Particular/DocsEngine/pull/653